syft/generic/pointers/object_pointer.py

```diff
@@ -41,45 +37,6 @@ class ObjectPointer(AbstractSendable, SyftSerializable):
     in the BaseWorker object in self.location.
     """
 
-    def __init__(
-        self,
-        location: "BaseWorker" = None,
-        id_at_location: Union[str, int] = None,
-        owner: "BaseWorker" = None,
-        id: Union[str, int] = None,
-        garbage_collect_data: bool = True,
-        point_to_attr: str = None,
-        tags: List[str] = None,
-        description: str = None,
-    ):
-        """Initializes a ObjectPointer.
-
-        Args:
-            location: An optional BaseWorker object which points to the worker
-                on which this pointer's object can be found.
-            id_at_location: An optional string or integer id of the object
-                being pointed to.
-            owner: An optional BaseWorker object to specify the worker on which
-                the pointer is located. It is also where the pointer is
-                registered if register is set to True. Note that this is
-                different from the location parameter that specifies where the
-                pointer points to.
-            id: An optional string or integer id of the ObjectPointer.
-            garbage_collect_data: If true (default), delete the remote object when the
-                pointer is deleted.
-            point_to_attr: string which can tell a pointer to not point directly to\
-                an object, but to point to an attribute of that object such as .child or
-                .grad. Note the string can be a chain (i.e., .child.child.child or
-                .grad.child.child). Defaults to None, which means don't point to any attr,
-                just point to then object corresponding to the id_at_location.
-        """
-        super().__init__(id=id, owner=owner, tags=tags, description=description)
-
-        self.location = location
-        self.id_at_location = id_at_location
-        self.garbage_collect_data = garbage_collect_data
-        self.point_to_attr = point_to_attr
-
```

Comment:
I think the `wrap` method below should be moved to `PointerTensor` since it is designed for tensors in mind. When we inherit directly from `ObjectPointer`, we do not want it to be a pointer to a tensor.

Reply:
In SyferText. Although it does not raise any error.

Reply:
Hmm, yeah, that's a good point. I've been mostly ignoring wrappers for the moment, because the series of changes I've been working on this week is intended to make it easier to remove wrappers altogether.

Reply:
Oh, if wrappers are going to disappear soon, we might keep it for now, not to waste time on something that is going to disappear anyway in a later PR. It is not breaking or anything for SyferText